core/graph/src/main/java/uk/gov/gchq/gaffer/graph/GraphConfig.java

```diff
@@ -64,6 +65,9 @@
     private String description;
     private List<GraphHook> hooks = new ArrayList<>();
 
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    private boolean skipDefaultSecurityHooks = false;
```

Comment:
I'm not sure about this.

Reply:
This makes sure that any existing graph config which doesn't have the field set will have the value set to false. Is there something I overlooked?

Reply:
I am just not sure whether having a boolean to toggle security hooks on/off is the best way of doing it. Although I can't quickly think of a better way. For example, OperationAuthoriser and OperationChainLimiter are also security related hooks.